packages/api/__tests__/utility.ts

```diff
@@ -96,6 +96,7 @@ export async function createGraphQLTestClientWithMongoDB(): Promise<TestClient>
   const mediaAdapter: KarmaMediaAdapter = {
     url: new URL('https://fakeurl.com'),
     token: 'fake',
+    internalURL: new URL('https://internalurl.com'),
```

Comment:
I don't think this is needed. `KarmaMediaAdapter` should work well without an internal URL.

Reply:
It shows an error when I remove it.